atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -51,49 +44,89 @@ class SendMailPlugin(ExitPlugin):
     # symbolic constants for states
     MANUAL_SUCCESS = 'manual_success'
     MANUAL_FAIL = 'manual_fail'
+    MANUAL_CANCELED = 'manual_canceled'
     AUTO_SUCCESS = 'auto_success'
     AUTO_FAIL = 'auto_fail'
     AUTO_CANCELED = 'auto_canceled'
-
-    allowed_states = set([MANUAL_SUCCESS, MANUAL_FAIL, AUTO_SUCCESS, AUTO_FAIL, AUTO_CANCELED])
-
-    PDC_TOKEN_FILE = 'pdc.token'
-    PDC_CONTACT_ROLE = 'Devel_Owner'
-
-    def __init__(self, tasker, workflow, send_on=None, url=None, submitter='unknown', pdc_url=None,
-                 pdc_verify_cert=True, pdc_component_df_label="com.redhat.component", pdc_secret_path=None,
-                 pdc_contact_role=None, smtp_uri=None, from_address=None,
-                 error_addresses=None):
+    DEFAULT_SUBMITTER = 'Unknown'
+
+    allowed_states = set([MANUAL_SUCCESS, MANUAL_FAIL, MANUAL_CANCELED,
+                          AUTO_SUCCESS, AUTO_FAIL, AUTO_CANCELED])
+
+    def __init__(self, tasker, workflow,
+                 send_on=None,
+                 url=None,
+                 smtp_host=None,
```

Comment:
its required, but there's no universal default for it. OSBS Client would remove a plugin if `smtp_host` or `from_address` is not set.

Should we keep this as a `None` or set some fake host name?